_layouts/post.liquid

```diff
@@ -13,7 +13,7 @@
         <main class="blog">
           <article>
             <h1>{{ title }}</h1>
-            <em>{{ page.date | dateFormat: "d LLLL yyyy" }}</em>
+            <em>{{ page.date | dateFormat: "d LLLL yyyy" }}</em> - by <b>{{ author }}</b>
```

Comment:
I think the reverse will read better
```suggestion
            by {{ author }}  — <em>{{ page.date | dateFormat: "d LLLL yyyy" }}</em>
```

Reply:
Sure I will change it so the author comes first.